insights/specs/default.py

```diff
@@ -52,19 +52,13 @@ def _make_rpm_formatter(fmt=None):
             '"buildhost":"%{BUILDHOST}"',
             '"sigpgp":"%{SIGPGP:pgpsig}"'
         ]
-
-    def inner(idx=None):
-        if idx:
-            return "\{" + ",".join(fmt[:idx]) + "\}\n"
-        else:
-            return "\{" + ",".join(fmt) + "\}\n"
-    return inner
-
-
-format_rpm = _make_rpm_formatter()
+    return "\{" + ",".join(fmt) + "\}\n"
 
 
 class DefaultSpecs(Specs):
+    # Variables that need to be pre-loaded
+    rpm_format = _make_rpm_formatter()
```

Comment:
@xiangce could you move this under the below deps specs section? I meant for that section to be for these types of specs that are essentially just variables used by real specs.